libzeth/tests/serialization/proto_utils_test.cpp

```diff
@@ -49,7 +49,24 @@ TEST(ProtoUtilsTest, PrimaryInputsEncodeDecode)
     ASSERT_EQ(inputs, inputs_decoded);
 }
 
-// TODO: Add test for accumulation_vector_from_string
+TEST(ProtoUtilsTest, AccumulationVectorEncodeDecode)
+{
+    libsnark::accumulation_vector<G1> acc_vect;
+    acc_vect.first = G1::random_element();
+    std::vector<G1> rest{G1::random_element(), G1::random_element()};
+    acc_vect.rest = libsnark::sparse_vector<G1>(std::move(rest));
+
+    std::string acc_vect_string =
+        libzeth::accumulation_vector_to_string<ppT>(acc_vect);
+
+    const libsnark::accumulation_vector<G1> acc_vect_decoded =
+        libzeth::accumulation_vector_from_string<ppT>(acc_vect_string);
+
+    std::string acc_vect_string_recovered =
```

Comment:
This variable does not seem to be used anywhere

Reply:
cleaned the function